packages/orchestration/src/exos/orchestrator.js

```diff
@@ -33,7 +28,7 @@ const trace = makeTracer('Orchestrator');
 
 /** @see {Orchestrator} */
 export const OrchestratorI = M.interface('Orchestrator', {
-  getChain: M.call(M.string()).returns(Vow$(ChainInfoShape)),
+  getChain: M.call(M.string()).returns(Vow$(M.remotable())),
```

Comment:
`M.remotable()` takes an optional `string` parameter. It can be very useful in debugging.

```suggestion
  getChain: M.call(M.string()).returns(Vow$(M.remotable('ChainFacade'))),
```